internal/dlog/dlog_test.go

```diff
@@ -30,19 +31,21 @@ type params struct {
 	p, order, g *big.Int
 }
 
-func get_params() *params {
-	p, _ := emmy.GetSafePrime(20)
-	pMin1 := new(big.Int).Sub(p, big.NewInt(1))
-	x := emmy.GetRandomInt(p)
+func get_params(t *testing.T) *params {
```

Comment:
Could you rename this as `getParams` since the Go convention is to use camelCase?

Reply:
Done.